tests/integration/defs/disaggregated/test_disaggregated.py

```
@@ -2103,6 +2108,198 @@ def test_disaggregated_cancel_large_context_requests(disaggregated_test_root,
+                    st_tokens, st_logprobs = \
+                        await collect_streaming_logprobs(resp, api_type)
+
+                assert ns_tokens == st_tokens, (
+                    f"[{api_type}] streaming vs non-streaming tokens mismatch")
+                assert len(ns_logprobs) == len(st_logprobs), (
+                    f"[{api_type}] logprobs length: "
+                    f"{len(ns_logprobs)} vs {len(st_logprobs)}")
+                # Skip position 0: the first token logprob can diverge
+                # between streaming and non-streaming in disaggregated mode
+                # due to the context/generation handoff boundary.
+                for i, (n, s) in enumerate(zip(ns_logprobs, st_logprobs)):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)